src/helpers/ValidateHelper.js

```diff
@@ -35,6 +35,9 @@ const isAccountNameError = (value, allowShort) => {
 		return `${suffix} be shorter then 63 symbols.`;
 	}
 
+	if (!/[.\-/0-9]/.test(value) || !(value.match(/[.\-/0-9aeiouy]/ig).length !== value.length)) {
```

Comment:
we probably should use cheap name check as different condition check with different error, for example below this check